lib/autoprefixer.js

```diff
@@ -58,6 +58,8 @@ module.exports = postcss.plugin('autoprefixer', (...reqs) => {
 
     if (options.browsers) {
         reqs = options.browsers;
+    } else {
```

Comment:
Why we need this change?

Reply:
If we want have `browsers` in `options`.

Reply:
But we don’t want to change API ;)

Reply:
I'm not sure how check setting `browsers` without getting it from `options` after initialize.

Reply:
Why you even need it? You improve tests, yeap? Not a API?

Reply:
Ouh, you mean that you need to get target browsers for tests? Changing original `options`  object is a bad way to do it.

You can use `info()` or check output prefixes.

Reply:
So weird. We have specified the browsers but can't get him by `instance.options`

Reply:
Or you can add special `plugin.browsers` with browsers after `browserslist`